pages/Technical-Gadgets.md

```diff
@@ -0,0 +1,42 @@
+The Technical Gadgets category has a variety of gadgets that are useful when playing Slimefun.  
+
+## Table of contents
+* [Electric Jetpack - I](https://github.com/Slimefun/Slimefun4/wiki/Jetpacks)
+* [Electric Jetpack - II](https://github.com/Slimefun/Slimefun4/wiki/Jetpacks)
+* [Electric Jetpack - III](https://github.com/Slimefun/Slimefun4/wiki/Jetpacks)
+* [Electric Jetpack - IV](https://github.com/Slimefun/Slimefun4/wiki/Jetpacks)
+* [Electric Jetpack - V](https://github.com/Slimefun/Slimefun4/wiki/Jetpacks)
+* [Electric Jetpack - VI](https://github.com/Slimefun/Slimefun4/wiki/Jetpacks)
+* [Electric Jetpack - VII](https://github.com/Slimefun/Slimefun4/wiki/Jetpacks)
+* [Parachute](https://github.com/Slimefun/Slimefun4/wiki/Technical-Gadgets#parachute)
+* [Hologram Projector](https://github.com/Slimefun/Slimefun4/wiki/Hologram-Projector)
+* [Multi Tool - I](https://github.com/Slimefun/Slimefun4/wiki/Multi-Tools)
+* [Multi Tool - II](https://github.com/Slimefun/Slimefun4/wiki/Multi-Tools)
+* [Multi Tool - III](https://github.com/Slimefun/Slimefun4/wiki/Multi-Tools)
+* [Multi Tool - IV](https://github.com/Slimefun/Slimefun4/wiki/Multi-Tools)
+* [Multi Tool - V](https://github.com/Slimefun/Slimefun4/wiki/Multi-Tools)
+* [Multi Tool - VI](https://github.com/Slimefun/Slimefun4/wiki/Multi-Tools)
+* [Multi Tool - VII](https://github.com/Slimefun/Slimefun4/wiki/Multi-Tools)
+* [Solar Helmet](https://github.com/Slimefun/Slimefun4/wiki/Technical-Gadgets#solar-helmet)
+* [Night Vision Goggles](https://github.com/Slimefun/Slimefun4/wiki/Technical-Gadgets#night-vision-goggles)
+* [Jet Boots - I](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Jet Boots - II](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Jet Boots - III](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Jet Boots - IV](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Jet Boots - V](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Jet Boots - VI](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Jet Boots - VII](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Armored Jet Boots](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Multimeter](https://github.com/Slimefun/Slimefun4/wiki/Technical-Gadgets#multimeter)
+
+### Parachute
+The Parachute can be used to slowly descent when you're in the air. You need to wear it and crouch for it to activate.
+
+### Solar Helmet
+The Solar Helmet when worn will charge any electric armor and anything electric in the player's hands.
+
+### Night Vision Goggles
+When worn, this piece of head armour gives you night vision.
```

Comment:
We should really standardize a language here. Do we want British English or American English?